core/controllers/editor_test.py

```diff
@@ -1153,6 +1482,62 @@ def test_transfering_ownership_to_the_community(self):
 
         self.logout()
 
+    def test_get_with_invalid_version_raises_error(self):
+        self.login(self.OWNER_EMAIL)
+        exp_id = 'exp_id'
+        self.save_new_valid_exploration(exp_id, self.owner_id)
+        self.get_json(
+            '%s/%s' % (feconf.EXPLORATION_DATA_PREFIX, exp_id),
+            params={'v': 'invalid_version'}, expected_status_int=404)
+        self.logout()
+
+    def test_put_with_invalid_new_member_raises_error(self):
+        self.login(self.OWNER_EMAIL)
+        exp_id = 'exp_id'
+        self.save_new_valid_exploration(exp_id, self.owner_id)
+        response = self.get_html_response('/create/%s' % exp_id)
+        csrf_token = self.get_csrf_token_from_response(response)
+        exploration = exp_services.get_exploration_by_id(exp_id)
+        with self.assertRaisesRegexp(
```

Comment:
Shouldn't this (and the following) have something that uses expected_status_int and response['error'], like all the other tests? We don't want to surface 500s to the user.

Reply:
Done